modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/integration.dm

```diff
@@ -53,6 +55,7 @@
 		. += ""
 		. += "Blood Drank: [bloodsuckerdatum.total_blood_drank]"
 		. += "Frenzy blood threshold: [bloodsuckerdatum.frenzy_enter_threshold()]"
+		. += "Normal Blood Cap: [bloodsuckerdatum.max_blood_volume]"
```

Comment:
Please don't add this as my PR already does

Reply:
already done as of [Legal Advice Pog](https://github.com/Bubberstation/Bubberstation/pull/1189/commits/1aa080a148a2787bcf0625bb7f64bfc811e7cbd7)